Assets/Scripts/Player/PlayerInventory.cs

```diff
@@ -58,19 +60,39 @@ void Start()
 	// Update is called once per frame
 	void Update()
 	{
+		_altScrollButton.Update();
 		CheckScroll();
 	}
 
 	void CheckScroll()
 	{
-		float scrollAmount = Input.GetAxis( "Scroll" + PlatformUtils.platformName );
-		if ( ( scrollAmount > WadeUtils.SMALLNUMBER || scrollAmount < -WadeUtils.SMALLNUMBER ) & heldResources.Count > 0 )
+		if ( heldResources.Count <= 1 )
 		{
-			// Need to do this so >0 rounds up and <0 rounds down
-			int nextIndex = resourceIndex + Mathf.Clamp( Mathf.RoundToInt( scrollAmount ), -1, 1 );
-			resourceIndex = MathUtils.Mod( nextIndex, heldResources.Count );
-			SpawnResourceObject();
+			// Nothing to scroll
+			return;
+		}
+
+		float scrollAmount = InputUtils.GetAxis("Scroll");
+
+		if ( scrollAmount < WadeUtils.SMALLNUMBER && scrollAmount > -WadeUtils.SMALLNUMBER )
```

Comment:
I think there's an `IsZero()` method for floats in `MathUtils`. If not, now would be a good time to add one.